### PR TITLE
Changes to standard output and Slack messages

### PR DESCRIPTION
**Pull request by sharkymcdongles**
This adds version to the standard output to show what version is being installed/upgraded.

This improves slack output to not use pretext to prevent it from being cut off.

### CODE REVIEW

**Comment by sharkymcdongles** [comment]
I think this is to a point where it can be reviewed.

File: internal/app/utils.go [hunk 1]
```diff
@@ -374,22 +374,33 @@ func deleteFile(path string) {
...
 		color = "#FF0000" // red
 	}
 
+	var contentBold string
 	var pretext string
+
 	if content == "" {
-		pretext = "No actions to perform!"
+		pretext = "*No actions to perform!*"
+		contentBold = content
```
Comment (luisdavim) [inline internal/app/utils.go]: Nti: At this point both variables are empty strings,  this line seems redundant.

**Comment by luisdavim** [comment]
Why make everything bold?

**Comment by sharkymcdongles** [comment]
> Why make everything bold?

The original code used the pretext field for displaying the plan, and pretext is bold by design. I just assumed you'd want to mirror the same bolding that was there. I can change it to whatever you guys think is best. I just extended what was there with no real plan except mirroring the original functionality best I could while adding some stuff. Please let me know what formatting you prefer.

As for the redundant bit, I will commit changes tomorrow to remedy that mistake on my part.

Thanks for the reply!

File: internal/app/utils.go (hunk 1 above)
Comment (sharkymcdongles) [reply internal/app/utils.go]: I went ahead and removed it as suggested.

**Pull request merged**